moonbeam-types-bundle/package-lock.json

```diff
@@ -1,98 +1,314 @@
 {
```

Comment:
you've told me before, but why do we commit the package-lock.json at all? Isn't is generated by building the package?

Reply:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
I see, it is necessary for a deterministic install. I guess builds are not deterministic without the `.lock` file.